django/contrib/admin/static/admin/css/forms.css

```diff
@@ -303,8 +303,10 @@ body.popup .submit-row {
     padding: 10px 15px;
     height: 15px;
     line-height: 15px;
-    margin: 0 0 0 5px;
+    margin: 0 5px 5px 5px;
     color: var(--button-fg);
+    vertical-align: top;
+    text-decoration: none;
```

Comment:
I would move `text_decoration` below to `focus` and `hover` styles:
```css
.submit-row a.deletelink:focus,
.submit-row a.deletelink:hover,
.submit-row a.deletelink:active {
    background: var(--delete-button-hover-bg);
    text-decoration: none;
}

.submit-row a.closelink:focus,
.submit-row a.closelink:hover,
.submit-row a.closelink:active {
    background: var(--close-button-hover-bg);
    text-decoration: none;
}
```

Reply:
Thank you for reviewing. :) Your point is excellent. 

Some information, why styles in the `base.css` wasn't work with cancel button: https://developer.mozilla.org/en-US/docs/Web/CSS/:visited#privacy_restrictions

`base.css`:
```css
a:link, a:visited {
    color: var(--link-fg);
    text-decoration: none;
    transition: color 0.15s, background 0.15s;
}
```